### ISSUE TITLE
The detection score missing sigmoid activation

### ISSUE THREAD

**Issue by fuqianya** [IDT-ITI/MMFusion-IML#5]
Hi,

Thank you for your excellent work! I noticed that the detection score in your code is trained using `torch.nn.BCEWithLogitsLoss`. To ensure the detection score falls within the range [0, 1], we typically apply a sigmoid function to the detection score. However, it seems that this crucial function is not implemented in the repository’s code. Could you please verify this?

**Comment by kostino** [comment]
Hello, 
torch.nn.BCEWithLogitsLoss accepts logits (and not outputs of sigmoid in the [0,1] range) as inputs and computes the sigmoid internally for numerical stability reasons. This means that for inference, you need to apply the sigmoid function to the model's outputs.

**Comment by fuqianya** [comment]
Yes! However, you have not applied the sigmoid function to the model's outputs in your code.

**Comment by kostino** [comment]
I had the wrong file version for the detection test. 33c1730d35d8a7dd9b80d42d48e982c6748dfcbd fixes it. Thanks for catching that!
